logfile_tests/reader_tests/reader_base_test.py

```diff
@@ -0,0 +1,151 @@
+"""
+Module contains tests for ReaderBase
+"""
+
+import pytest
+from logfile.readers.reader_base import ReaderBase
+
+# pylint: disable=redefined-outer-name
+
+
+class MockReaderBase(ReaderBase):
```

Comment:
Consider using unittest.mock instead.

Reply:
Did not know that existed, I will look into that for the next pull request